File: C#/백준/Bronze/31306. Is Y a Vowel？/Is Y a Vowel？.cs
using System;

namespace q{
    class w{
        static void Main(){
            string a = Console.ReadLine();
            int cnt1 = 0;
            int cnt2 = 0;
            for (int i =0; i<a.Length; i++){
                if(a[i]=='a' || a[i]=='e'||a[i]=='i'||a[i]=='o'||a[i]=='u'){
                    cnt1++;
                }
                else if(a[i]=='y'){
                    cnt2++;
                }
            }
            Console.WriteLine($"{cnt1} {cnt1+cnt2}");
        }
    }
}

File: Java/백준/Bronze/31306. Is Y a Vowel？/Is Y a Vowel？.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        String a = in.next();
        int cnt1 = 0;
        int cnt2 = 0;
        for(int i=0; i<a.length(); i++){
            char b = a.charAt(i);
            if(b=='a' || b=='e' || b=='i' || b=='o'|| b=='u')cnt1++;
            else if(b=='y')cnt2++;
        }
        System.out.printf("%d %d",cnt1,cnt1+cnt2);
    }
}